routing/router.go

```diff
@@ -366,8 +366,23 @@ func (a alice) then(h http.Handler) http.Handler {
 func buildHttpHandlerLogger(route Route, handlerFunction http.Handler) http.Handler {
 	c := alice{}
 	c = c.append(hlog.NewHandler(zerolog.With().Str("route", route.Name).Logger()))
-	c = c.append(hlog.HostHandler("Host", true))
-	c = c.append(hlog.RequestIDHandler("RequestID", "Request-Id"))
+	c = c.append(hlog.HostHandler("host", true))
+	c = c.append(hlog.RemoteAddrHandler("remoteAddr"))
+
+	// extract and log the request ID - if we have a X-Request-Id header we use it; otherwise, we generate our own
+	xRequestIdHeader := "X-Request-Id"
+	xRequestFieldKey := "xRequestId"
```

Comment:
nit: can we keep this `RequestID`?

Reply:
We can. This can be anything we want. You want "RequestID" or "requestId" I was putting everything in our structured log context as `camelCase`

Reply:
I changed it to "requestId" in latest commit